_includes/endpoint_responses/delete_delete_transaction_transactionId.json

```diff
@@ -71,6 +71,7 @@
   "SendEmailNotifications": true,
   "CreatedDateTime": "2016-06-15T23:33:04.1965465+02:00",
   "ModifiedDateTime": "2016-06-15T23:33:04.1965465+02:00",
-  "CanceledDateTime": null,
+  "CanceledDateTime": "2016-06-15T23:34:15+02:00",
+  "CancelationReason": "This document is out-dated",
```

Comment:
I found 'Cancelled' already in the code (e.g. TransactionStatus.cs) which indicates the use of British English.
Best to keep it 1 form of English, so we should change _CanceledDateTime_ to _CancelledDateTime_.
Also, it might be good to mention this in the [Guidelines](https://evidos.atlassian.net/wiki/spaces/proces/pages/49547702/Development+richtlijnen+Guidelines) which I did already, but correct me if it should have been American English.

Reply:
Correct, unfortunately CanceledDateTime was already spelled incorrectly and we can't change this property without making a breaking API change.
To avoid confusion The CancelationReason is using the same spelling as they are both referring to the same property.